tools/cloud_usage.rb

```diff
@@ -77,14 +77,32 @@ def run
         end
       end
       # internal openstack
-      command :"upshift" do |c|
+      command :"psi" do |c|
```

Comment:
@pruan-rht   There are PSI OCP and PSI OSP.   `upshift = PSI OSP`

Reply:
@liangxia perhaps just openstack would be better then because that's what the backend is

Reply:
We also have several OpenStack there.  How about rhos-d ?